src/app/project/contact-form/contact-form.component.scss

```diff
@@ -0,0 +1,48 @@
+.successful-send {
```

Comment:
I've been guilty forgetting this but I think if you use the `git mv` command, you can tell git that a file was moved instead of being deleted and recreated.

Reply:
I also changed the name of the file because it went from CSS to SCSS so I don't think move would work in this instance.